### PR TITLE
Complete Homework №3.2

### PR DESCRIPTION
**Pull request by SorokinTim**

### CODE REVIEW

File: src/models/User.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+User.init({
+    id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        allowNull: false,
+    },
+    isDeleted: {
+        type: DataTypes.BOOLEAN,
+        allowNull: false,
+    },
```
Comment (potsiahevich-dmitry) [inline src/models/User.ts:14]: This field is redundant now. You can remove it and use the real deleting

File: src/services/users/userService.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import { v4 as uuid } from "uuid";
+import { Op, literal } from "sequelize";
+import sequelize from "../../data-acess/db";
+import { User as UserModel } from "../../models/User";
+import { User, UserInternalProps } from "../../types/user";
+
+export default class UserService {
+    async getUserById(id: string) {
+        await sequelize.sync();
```
Comment (potsiahevich-dmitry) [inline src/services/users/userService.ts:9]: Here is no sense to sync the db on any call. It makes your performance to be worse. The sync should be done once after all models initialization or sync models only if it's supposed to be not initialized all at once

File: src/services/users/userService.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        if (!currentUser) {
+            return;
+        }
+
+        await currentUser.update({
+            login,
+            password,
+            age,
+        });
+        await currentUser.save();
```
Comment (potsiahevich-dmitry) [inline src/services/users/userService.ts:42]: method update already saves the data in db, so there is no sense to call save

**Review by potsiahevich-dmitry — APPROVED** [review]
Fix the comments in the next task

**Comment by potsiahevich-dmitry** [comment]
Please, resolve the conflicts and merge this PR into master

File: src/models/User.ts (hunk 1 above)
Comment (SorokinTim) [reply src/models/User.ts]: Fixed into https://github.com/SorokinTim/nodejs-gmp/pull/6/commits/e45fdbc193c750f092f42a6891b7d1d2c927945b commit

File: src/services/users/userService.ts (hunk 2 above)
Comment (SorokinTim) [reply src/services/users/userService.ts]: Fixed into https://github.com/SorokinTim/nodejs-gmp/pull/6/commits/a3a56e47c667ff530cd567600bbbf06ec3e0f01d commit

File: src/services/users/userService.ts (hunk 3 above)
Comment (SorokinTim) [reply src/services/users/userService.ts]: Fixed into https://github.com/SorokinTim/nodejs-gmp/pull/6/commits/a016fe24a80382fa01f5a13b7bf7a8b8ba161166 commit

**Pull request merged**